Description: remove text-unidecode dependency
 unidecode was replaced by text-unidecode in 3.0.2. text-unidecode is
 not packaged in Debian yet and has the same interface as unidecode.
 Since both can be used interchangeably, use unidecode instead of
 text-unidecode.
Author: Hugo Lefeuvre <hle@debian.org>
Bug-Debian: http://bugs.debian.org/933242
Forwarded: not-needed
--- a/setup.py	2019-11-02 11:12:30.964071073 +0100
+++ b/setup.py	2019-11-02 11:13:00.119830799 +0100
@@ -14,7 +14,7 @@
 author = 'Val Neekman'
 author_email = 'info@neekware.com'
 license = 'MIT'
-install_requires = ['text-unidecode>=1.3']
+install_requires = []
 extras_require = {'unidecode': ['Unidecode>=1.1.1']}
 
 classifiers = [
